lib/brine/step_definitions/request_construction.rb

```diff
@@ -12,6 +13,10 @@
   add_header(header, value)
 end
 
+When(/^the request auth basic header `([^`]*)` is assigned `([^`]*)`$/) do |header, value|
```

Comment:
if doing basic auth then the arguments should be the username and password. The phrasing of the step should be considered relative to other steps...this step in particular is very confusing since `auth` is in the step but the header is an argument so the step could read very different than what is being done.

A better alternative may be something like:
```
When the request credentials are set for basic auth user `$user` and password `$password`
```
This also needs be accompanied by appropriate documentation and specification updates.